packages/server/src/api/routes/static.ts

```diff
@@ -51,8 +51,8 @@ router
     controller.deleteObjects
   )
   .get("/app/preview", authorized(BUILDER), controller.serveBuilderPreview)
-  .get("/:appId/:path*", controller.serveApp)
   .get("/app/:appUrl/:path*", controller.serveApp)
+  .get("/:appId/:path*", controller.serveApp)
```

Comment:
This was kind of weird, I don't know if this is a typescript thing, with sending two routes with different potential query params to the endpoint, but changing the order of these routes resulted in different values being available in `ctx` in the controller.

Putting them in this order let me check if appUrl was provided, and hence if a published app was being rendered.

Reply:
If you're referring to the ordering of the matchers, yes I'm aware, I don't see how the explains the issue. 

The paths are qualified such that I don't see how how both `/app/some_app_url` and `/app_some app_url` would both just match the first one and ignore the second, unless for some reason the `/` characters are matching anything.

Unless you mean two matchers can't point to the same controller function? But I don't really see how that explains the issue either.